### PR TITLE
[BD-35] fix: update variables and override progress-bar

### PR DESCRIPTION
**Pull request by jignaciopm**
@abutterworth 
update variables and override progress-bar

Captura de pantalla de 2021-04-13 14-15-57
Captura de pantalla de 2021-04-13 14-16-04

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @jignaciopm! I've created [OSPR-5736](https://openedx.atlassian.net/browse/OSPR-5736) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by natabene** [comment]
@jignaciopm Thank you for your contribution. @abutterworth Would you like to review this?

File: src/ProgressBar/ProgressBar.scss [hunk 1]
```diff
@@ -1,2 +1,10 @@
 @import "variables";
 @import "~bootstrap/scss/progress";
+
+.progress { 
+    border: 1px solid $gray-500;
+}
+.progress-bar { 
+    background-color: $accent-a;
+}
```
Comment (abutterworth) [inline src/ProgressBar/ProgressBar.scss:9]: Please use `$progress-bar-bg` and `$progress-bar-border-width` in these definitions

**Review by abutterworth — CHANGES_REQUESTED** [review]

File: src/ProgressBar/ProgressBar.scss (hunk 1 above)
Comment (jignaciopm) [reply src/ProgressBar/ProgressBar.scss]: @abutterworth fixed and added a new variable $progress-bar-border-color to store the border color.

**Comment by natabene** [comment]
@jignaciopm Please make sure to include [BD-35] tag into PR titles in the future, if you are contributing work for the BD-35 Blended project. Otherwise they won't be processed properly by our bots.

**Review by abutterworth — APPROVED** [review]

**Pull request merged**